pubsublite-spark-sql-streaming/src/main/java/com/google/cloud/pubsublite/spark/PslContinuousReader.java

```diff
@@ -108,6 +113,14 @@ public StructType readSchema() {
         .map(
             v ->
                 new PslContinuousInputPartition(
+                    (consumer) ->
```

Comment:
Can you push this up one more level? Have PslContinuousReader take a SubscriberFactory.

Reply:
looks like all it needs from this class is the partition. So can you make an interface instead that has

Subscriber newSubscriber(Partition partition, Consumer<List<SequencedMessage>> messages)

and is serializable?

Reply:
done.